### ISSUE TITLE
No new accessory is shown in Home app

### ISSUE THREAD

**Issue by daciand** [NorDroN/homebridge-wideq#15]
Hi,
I have just installed your plugin, restarted homebridge and these are the homebridge log entries related to this plugin:
`
Jun 20 09:46:09 raspberrypi homebridge[22043]: [6/20/2020, 09:46:09] [WideQ] Initializing WideQ platform...
Jun 20 09:46:09 raspberrypi homebridge[22043]: [6/20/2020, 09:46:09] [WideQ] **************************************************************
Jun 20 09:46:09 raspberrypi homebridge[22043]: [6/20/2020, 09:46:09] [WideQ] WideQ v0.1.9 by NorDroN
Jun 20 09:46:09 raspberrypi homebridge[22043]: [6/20/2020, 09:46:09] [WideQ] GitHub: https://github.com/NorDroN/homebridge-wideq
Jun 20 09:46:09 raspberrypi homebridge[22043]: [6/20/2020, 09:46:09] [WideQ] Email: nordron@live.ru
Jun 20 09:46:09 raspberrypi homebridge[22043]: [6/20/2020, 09:46:09] [WideQ] **************************************************************
Jun 20 09:46:09 raspberrypi homebridge[22043]: [6/20/2020, 09:46:09] [WideQ] [INFO] Create client from token and get devices
`

Here is my plugin configuration:
`
{
    "platform": "WideQ",
    "refresh_token": "[token_taken_from_wideq_tool]"
    "country": "RO",
    "language": "ro-RO",
    "debug": true,
    "interval": 10
},
`

My problem is that there is no visible LG accessory that could be added in Home app.
Could it be that while wideq-state.json uses "oauthRoot":"https://ro.lgeapi.com", the URL returned from the browser contains "&oauth2_backend_url=https://gb.lgeapi.com/"?
I've launched wideq tool with the following arguments:
`
pi@raspberrypi:~ $ wideq auth --country RO --language ro-RO
Log in here: https://ro.m.lgaccount.com/login/sign_in?country=RO&language=ro-RO&svcCode=SVC202&authSvr=oauth2&client_id=LGAO2                                                                                  21A02&division=ha&grant_type=password
`

Please help :)

**Comment by NorDroN** [comment]
Have you been authorized successful? Did you get refresh_token?
Try to run `wideq ls -c RO -l ro-RO -t [refresh_token here]`.
What do you see?

**Comment by daciand** [comment]
Yes, everything was fine at that step.
I just ran `pi@raspberrypi:~ $ wideq ls -c RO -l ro-RO -t [my_refresh_token]` but here is no output:
`
pi@raspberrypi:~ $ wideq ls -c RO -l ro-RO -t dcf2...ed08
pi@raspberrypi:~ $
`

**Comment by u20p17** [comment]
Hi,
I now have successfully recieved my refesh token and when I start homebridge it seems that everything is ok. But I can also see no new device. 
I would like to add my washer ( F16WD128GH). What could be the problem?

Thanks in advance!

**Comment by LeviJSmith** [comment]
Having same issue.  Did u ever get this resolved?

**Comment by daciand** [comment]
No, I have abandoned until we'll get a reply for this ticket. I'm sorry but I don't have any idea what I could do any further in order to get this issue fixed or at least find out more information about it.

**Comment by u20p17** [comment]
What information would you need? I can test thinge with my washing maschine for you! :)

**Comment by daciand** [comment]
Do you have any idea how could I activate debug logging for this plugin? I already have `"debug": true` set but it doesn't produce any debug output, other than:
```
Aug 11 12:16:28 raspberrypi homebridge[10593]: [8/11/2020, 12:16:28] Loaded plugin: homebridge-wideq@0.1.9
Aug 11 12:16:28 raspberrypi homebridge[10593]: [8/11/2020, 12:16:28] Registering platform 'homebridge-wideq.WideQ'
Aug 11 12:16:28 raspberrypi homebridge[10593]: [8/11/2020, 12:16:28] [WideQ] Initializing WideQ platform...
Aug 11 12:16:28 raspberrypi homebridge[10593]: [8/11/2020, 12:16:28] [WideQ] **************************************************************
Aug 11 12:16:28 raspberrypi homebridge[10593]: [8/11/2020, 12:16:28] [WideQ] WideQ v0.1.9 by NorDroN
Aug 11 12:16:28 raspberrypi homebridge[10593]: [8/11/2020, 12:16:28] [WideQ] GitHub: https://github.com/NorDroN/homebridge-wideq
Aug 11 12:16:28 raspberrypi homebridge[10593]: [8/11/2020, 12:16:28] [WideQ] Email: nordron@live.ru
Aug 11 12:16:28 raspberrypi homebridge[10593]: [8/11/2020, 12:16:28] [WideQ] **************************************************************
Aug 11 12:16:29 raspberrypi homebridge[10593]: [8/11/2020, 12:16:29] [WideQ] [INFO] Create client from token and get devices

```

**Comment by 400HPMustang** [comment]
I'm getting the same thing, same debug information.

**Comment by adamrzanek** [comment]
I have the same problem :(

**Comment by chenqing212** [comment]
I have the same issue. I retrieved refresh token without issue and tested the toke without issue, but got the same debug info and just couldn’t see my device in homebridge.

I also tried different language setting, no luck. But as soon as I changed the country, it didn’t authenticate the token which means the token was specifically for the user with that specific locale setting.

Background with my LG account was, I had an gmail account with the LG fridge configured. But Gmail didn’t work with token retrieval, I had to de-couple the fridge with the account and registered a new email just to be able to sign up with LG.
